app/test/frontend/golden/pkg_versions_page.html

```diff
@@ -0,0 +1,114 @@
+
```

Comment:
Where does this come from? A new URL?

Reply:
A new test, we didn't have a golden template for /packages/[pkg]/versions